docs/specs/legacy.yml

```diff
@@ -410,10 +411,13 @@ inputs:
       docs/v1/: .
       docs/v2/: .
     monikerDefinition: monikerDefinition.json
-  docs/v1/TOC.md:
+  docs/v1/TOC.md: |
+    # [link](../c.md)
   docs/v1/f1/a.md:
-  docs/v2/TOC.md:
+  docs/v2/TOC.md: |
+    # [link](../c.md)
```

Comment:
Add a new case instead of changing the existing one, seems the existing one already covered this scenario

Reply:
reverted, and this case already cover file path ordering